python/examples/bert/benchmark_batch.py

```diff
@@ -35,19 +35,28 @@ def single_func(idx, resource):
     dataset = []
     for line in fin:
         dataset.append(line.strip())
+    profile_flags = False
+    if os.environ["FLAGS_profile_client"]:
+        profile_flags = True
     if args.request == "rpc":
         reader = BertReader(vocab_file="vocab.txt", max_seq_len=20)
         fetch = ["pooled_output"]
         client = Client()
         client.load_client_config(args.model)
         client.connect([resource["endpoint"][idx % len(resource["endpoint"])]])
-        feed_batch = []
-        for bi in range(args.batch_size):
-            feed_batch.append(reader.process(dataset[bi]))
-
         start = time.time()
         for i in range(1000):
             if args.batch_size >= 1:
+                feed_batch = []
+                b_start = time.time()
+                for bi in range(args.batch_size):
+                    feed_batch.append(reader.process(dataset[bi]))
+                b_end = time.time()
+                if profile_flags:
+                    print("PROFILE\tpid:{}\tbert+pre_0:{} bert_pre_1:{}".format(
+                        os.getpid(),
+                        int(round(b_start * 1000000)),
+                        int(round(b_end * 1000000))))
                 result = client.batch_predict(
```

Comment:
I think we can use predict instead of batch_predict here.

Reply:
changed to predict